src/reset-password/messages.js

```diff
@@ -36,6 +36,31 @@ const messages = defineMessages({
     defaultMessage: 'Password Reset Complete.',
     description: 'header message when reset is successful.',
   },
+  'forgot.password.confirmation.sign.in.link': {
+    id: 'forgot.password.confirmation.sign.in.link',
+    defaultMessage: 'sign-in',
+    description: 'link text used in message to refer to sign in page',
+  },
+  'reset.password.request.forgot.password.text': {
+    id: 'reset.password.request.forgot.password.text',
+    defaultMessage: 'Forgot Password',
+    description: 'Forgot password text',
+  },
+  'reset.password.request.invalid.token.header': {
+    id: 'reset.password.request.invalid.token.header',
+    defaultMessage: 'Invalid Password Reset Link',
+    description: 'Invalid password reset link help text heading',
+  },
+  'reset.password.empty.new.password.field.error': {
+    id: 'reset.password.empty.new.password.field.error',
+    defaultMessage: 'Please enter your New Password.',
```

Comment:
Are we intentionally keeping the first letters capital for `New Password`?

Reply:
yes to match the field name